examples/allennlp/allennlp_jsonnet.py

```diff
@@ -7,16 +7,7 @@
 Since it is too time-consuming to use the training dataset,
 we here use the validation dataset instead.
 
-We have the following two ways to execute this example:
-
-(1) Execute this code directly.
-    $ python allennlp_jsonnet.py
-
-
-(2) Execute through CLI.
-    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
-    $ optuna study optimize allennlp_jsonnet.py objective --n-trials=100 --study-name $STUDY_NAME \
-      --storage sqlite:///example.db
+$ python allennlp_jsonnet.py
```

Comment:
Hm, do you think we should keep this line(s)? I though we could remove it as well since it's a bit out-of-place without describing text.

Reply:
Hmm, I see as we are not really explaining or using it any further too. Sure, will do!

Reply:
Thanks for the quick fix. 

What do you think of doing that with the other examples as well that don't take any additional (pruning) options?

Reply:
Sure, so will just keep the exec command for the pruning options as its obvious for the others.

Reply:
Sounds good 👍